v2/yarpcprotobuf/protoc-gen-yarpc-go/internal/generator/registry.go

```diff
@@ -226,6 +276,81 @@ func (r *registry) newPackage(f *descriptor.FileDescriptorProto) *Package {
 	}
 }
 
+// getReflection returns the reflection details for the given FileDescriptorProto,
+// which includes the reflection variable name and the individual file's serialized
+// source representation.
+func getReflection(fd *descriptor.FileDescriptorProto) (*Reflection, error) {
```

Comment:
Yea that's true, but this isn't really a getter since it's not a method on a struct. This is resolving the `Reflection` type, given the `*descriptor.FileDescriptorProto`.

I'm fine with renaming these functions though. I'm just not sure what the name should be. I'd like to avoid using `load` since I want to continue to distinguish that for methods that mutate/populate the `registry`.

What are your thoughts?

Reply:
no worries, it's fine as is. I don't have a better suggestion :p